.github/labeler.yml

```
@@ -158,3 +158,9 @@ HTML:
 JavaScript:
   - changed-files:
       - any-glob-to-any-file: '**/*.js'
+
+# test suite
+testsuite:
+  - changed-files:
+      - any-glob-to-any-file:
+          - */*/testsuite/**
```

```suggestion
          - **/testsuite/**
```
If it isn't a problem to match a root folder named testsuite